docs/apm/transactions.asciidoc

```diff
@@ -2,54 +2,73 @@
-The *time spent by span type*, *transaction duration* and *requests per minute* chart display information on all transactions associated with the selected service.
-The *Transactions* table, however, provides only a list of _transaction groups_ for the selected service.
-In other words, this view groups all transactions of the same name together, and only displays one transaction for each group.
 
 [role="screenshot"]
 image::apm/images/apm-transactions-overview.png[Example view of transactions table in the APM app in Kibana]
 
-*Time spent by span type* -- Most agents support breakdown graphs in the APM app.
-This graph is an easy way to visualize where your application is spending most of its time.
-For example, is your app spending time in external calls, database processing, or application code execution?
+The *time spent by span type*, *transaction duration*, and *requests per minute* chart display information on all transactions associated with the selected service:
 
+*Time spent by span type*::
+Visualize where your application is spending most of its time.
+For example, is your app spending time in external calls, database processing, or application code execution?
++
 The time a transaction took to complete is also recorded and displayed on the chart under the "app" label.
 "app" indicates that something was happening within the application, but we're not sure exactly what.
 This could be a sign that the agent does not have auto-instrumentation for whatever was happening during that time.
-
++
 It's important to note that if you have asynchronous spans, the sum of all span times may exceed the duration of the transaction.
 
-*Transaction duration* shows the response times for this service and is broken down into average, 95th, and 99th percentile.
+*Transaction duration*::
+Response times for this service, broken down into average, 95th, and 99th percentile.
 If there's a weird spike that you'd like to investigate,
 you can simply zoom in on the graph - this will adjust the specific time range,
 and all of the data on the page will update accordingly.
 
-*Requests per minute* is divided into response codes: 2xx, 3xx, 4xx, etc.,
+*Requests per minute*::
+Visualize response codes: `2xx`, `3xx`, `4xx`, etc.,
 and is useful for determining if you're serving more of one code than you typically do.
 Like in the Transaction duration graph, you can zoom in on anomalies to further investigate them.
 
-The *Transactions* table is similar to the <<traces,traces>> overview and shows the name of each transaction occurring in the selected service.
-Transactions with the same name are grouped together and only shown once in this table.
+[[transactions-table]]
+==== Transactions table
+
+The *Transactions* table displays a list of _transaction groups_ for the selected service.
+In other words, this view groups all transactions of the same name together, and only displays one transaction for each group.
```

Comment:
Maybe this should be "one entry for each group"? The row displays aggregations for transactions for a specific transaction name.